test/serializer/spec_test.rb

```diff
@@ -1,70 +1,65 @@
 require "minitest_helper"
 
-module CollectionJson
-  module Spec
-    class TestSpec < Minitest::Test
-      include TestHelper
+class TestSpec < Minitest::Test
+  include TestHelper
 
-      def test_that_random_attributes_cannot_be_passed
-        serializer = empty_serializer_for(@user)
-        serializer.items.attributes = [
-          name: { unknown: "this should not be valid" }
-        ]
-        serializer.items.links = serializer.class._links = [
-          dashboard: {
-            href: "http://example.com",
-            unknown: "this should not be valid"
-          }
-        ]
-        serializer.class._template = [
-          name: { whatever: "this should not be valid" }
-        ]
-        serializer.class._queries = [
-          name: {
-            href: "http://example.com",
-            whatever: "this should be invalid",
-            data: [
-              unknown: "this should be invalid"
-            ]
-          }
+  def test_that_random_attributes_cannot_be_passed
```

Comment:
Assignment Branch Condition size for test_that_random_attributes_cannot_be_passed is too high. [65.01/15]